chalice/app.py

```diff
@@ -379,6 +429,20 @@ def _register_view(view_func):
             return view_func
         return _register_view
 
+    def _validate_route_content_types(self, name, content_types):
```

Comment:
Can we move this to the validation already in deployer.py?  I'd prefer to actually move validation out of app and into deployment (in general).  Along the same lines as the previous comment, we want to minimize overhead as much as possible, and we don't need to be doing validation every time the app is imported.